bindings/java/java/src/main/java/org/ethereum/evmc/Host.java

```diff
@@ -3,23 +3,17 @@
 // Licensed under the Apache License, Version 2.0.
 package org.ethereum.evmc;
 
-import static java.util.Objects.requireNonNull;
-
 import java.nio.ByteBuffer;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
 
 /**
  * The Host interface.
  *
  * <p>The set of all callback functions expected by VM instances.
  */
 final class Host {
-  static private ByteBuffer ensureDirectBuffer(ByteBuffer input) {
+  private static ByteBuffer ensureDirectBuffer(ByteBuffer input) {
     // Reallocate if needed.
-    if (!input.isDirect())
-      return ByteBuffer.allocateDirect(input.remaining()).put(input);
+    if (!input.isDirect()) return ByteBuffer.allocateDirect(input.remaining()).put(input);
```

Comment:
Are these stylistic changes done manually or part of some language linter?

Reply:
yes, it's using (spotless)[https://github.com/diffplug/spotless].

Might be better to use braces for if statements in java always.

Reply:
I thought we depend on this setting (https://github.com/ethereum/evmc/blob/master/bindings/java/build.gradle#L19-L31) to enforce out coding style.

Is CI not actually enforcing this? Are you using a different version of spotless?

Reply:
According to https://www.jvt.me/posts/2020/05/15/gradle-spotless/ we have to run `./gradlew clean spotlessApply test`, which we actually do: https://github.com/ethereum/evmc/blob/master/bindings/java/Makefile#L13

Reply:
Ah, I see we should be running `spotlessCheck` to find issues haha.

Reply:
Fixed in #581.